main.tf

```
@@ -24,14 +24,14 @@ resource "aws_db_instance" "default" {
 
   identifier            = module.this.id
   name                  = var.database_name
-  username              = var.database_user
-  password              = var.database_password
+  username              = length(var.replicate_source_db) > 0 ? null : var.database_user
+  password              = length(var.replicate_source_db) > 0 ? null : var.database_password
```

can we set the comparison so null is the second argument

```suggestion
  password              = length(var.replicate_source_db) == 0 ? var.database_password : null
```